src/current/cockroachcloud/serverless-cluster-management.md

```diff
@@ -98,6 +98,14 @@ To set the primary region:
 1. Select **Set primary region** next to your preferred region.
 1. Click **Update**.
 
+### Move cluster to a new region
+
+You currently cannot move a CockroachDB {{ site.data.products.serverless }} cluster from one region to another directly. Instead, you will need to back up your data and restore it into a new CockroachDB {{ site.data.products.serverless }} cluster with the desired region configuration: 
+
+1. [Back up your existing cluster's data]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}).
```

Comment:
I think you can also restore from a managed-service backup, yes?

Reply:
@mdlinville According to [this note](https://www.cockroachlabs.com/docs/cockroachcloud/use-managed-service-backups#restore-a-cluster), cross-cluster backups are not supported via the Cloud Console for serverless clusters.